features/step_definitions/networking.rb

```diff
@@ -631,3 +631,24 @@
     raise "Failed to delete the docker container." unless @result[:success]
   }
 end
+
+
+Given /^the multus is enabled on the cluster$/ do
+  ensure_admin_tagged
+
+  @result = admin.cli_exec(:get, resource: "daemonset", resource_name: "multus", n: "openshift-multus", template: "{{.status.desiredNumberScheduled}}")
```

Comment:
@bmeng, we can leverage off the existing `daemon_set` class to get these parameters which can shorten this step to 
```
  org_user = user
  org_project = project if @project
  project('openshift-multus')
  step %Q/I switch to cluster admin pseudo user/
  desired = daemon_set('multus').replica_counters[:desired]
  available = daemon_set('multus').replica_counters[:available]
  raise "Multus is not running correctly!" unless available == desired
  @user = org_user
  @project = org_project if org_project
```

Reply:
@bmeng the only downside is that we have to take care of project and user switching within the step.  It looks cleaner IMO, WDYT @akostadinov?

Reply:
I don't see the need for `I switch to cluster admin pseudo user` so restoring user shouldn'e be needed.
```
desired = daemon_set('multus').replica_counters(user: admin)[:desired]
```

Restoring project is not a big deal.

Reply:
@akostadinov, accessing this `openshift-multus` project needs admin privilege, so I think we need to switch the user since we can't guarantee the user has admin privilege prior to calling the method.

Reply:
hmm...actually, I may be wrong about the privilege...need to check it later.

Reply:
that's why I suggest `daemon_set('multus').replica_counters(user: admin)[:desired]`. This means query as cluster admin.

Reply:
ah, I didn't see that suggestion earlier... yeah, that looks cleaner and we can remove the switching part which results in only 3 lines (1 if we cramp everything inline)
```
desired = daemon_set('multus').replica_counters[:desired]
available = daemon_set('multus').replica_counters[:available]
raise "Multus is not running correctly!" unless available == desired
```
sweet!